run.sh

```diff
@@ -18,7 +18,7 @@ if [ -n "${INIT_BACKUP}" ]; then
 			sleep 1
 		done
 	fi
-	${BACKUP_CMD}
+	${BACKUP_CMD}  >> ${BACKUP_LOG} 2>&1
```

Comment:
This change introduced an error when running the super-linter, exactly the SHELL_SHFMT, test. Can you please remove the whitespace in front of the dollar sign?

Reply:
Looks like the double whitespace before the redirect needs also to be fixed.

Reply:
Looks much better now. Thanks.